.github/workflows/golangci-lint.yml

```diff
@@ -26,4 +26,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@v6
         with:
-          version: v1.61.0
+          version: v1.64.5
```

Comment:
There were nonsensical lint errors show. This is also available as a separate pr under #251